extension/awsecshealthcheckextension/extension.go

```diff
@@ -0,0 +1,133 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package awsecshealthcheckextension
+
+import (
+	"context"
+	"net"
+	"net/http"
+	"time"
+
+	"github.com/jaegertracing/jaeger/pkg/healthcheck"
+	"go.uber.org/zap"
+
+	"go.opencensus.io/stats/view"
+	"go.opentelemetry.io/collector/component"
+)
+
+type ecsHealthCheckExtension struct {
+	config   Config
+	logger   *zap.Logger
+	state    *healthcheck.HealthCheck
```

Comment:
```suggestion
```

This seems vestigial and unnecessary.  The only thing ever done with it is setting a logger.

Reply:
deleted

Reply:
It's still here.